packages/tools/src/index.ts

```diff
@@ -135,6 +136,7 @@ export {
   AngleTool,
   CobbAngleTool,
   UltrasoundDirectionalTool,
+  UltrasoundAnnotationTool,
```

Comment:
I think a more specific name makes more sense, let's ask the team to see what is suited.